frontend/components/elements/Button/Button.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+const button = {
+  width: '0.25em',
+  borderOpacity: 0.6,
+  colorText: '#FFFFFF',
+  bgColor: 'rgba(0, 0, 0, 0.2)',
+  hoverColor: '#0B0A0A',
+  hoverBgColor: '#FFFFFF',
+  disabledBgColor: 'rgba(0, 0, 0, 0.2)',
+  disabledColor: 'rgba(255,255,255, 0.7)',
+  borderColor: '255, 255, 255',
+};
+
+const StyledButton = styled('a')`
```

Comment:
I would oppose to naming `a` tag a `StyledButton` it introduces unnecessary confusion